### PR TITLE
Improvement: Slide behaviour for anchor

### PR DESCRIPTION
**Pull request by wutschel**
## Description

When the anchor is visible (partially displayed screen on the left or right), the anchor always opened to full view once touched. The user would expect that the anchor is snapping back to its position when the anchor is moved less than 50% of the screen width.

This PR does a bit of rework and refinement of the sliding conditions. Now the anchor's behaviour is same as for the fullscreen:
1. Flicking a fully visible screen moves it into an anchor
2. Moving a fully visible screen >50% moves it into an anchor
3. Moving a fully visible screen <50% lets it snap back to full screen
4. Flicking an anchor will switch to full screen
5. Moving an anchor >50% opens it to a full screen
6. Moving an anchor  <50% lets it snap back to the anchor

In addition, the PR removes the obsolete notification `"ECSLidingSwipeLeft"`. The condition to send this notification was only met in very rare cases, and the controllers handling it have an alternative `UISwipeGestureRecognizer` defined to handle the swipe gesture.

## Summary for release notes

Improvement: Slide behaviour for anchor

### CODE REVIEW

File: XBMC Remote/ECSlidingViewController/ECSlidingViewController.m [hunk 1]
```diff
@@ -293,6 +293,24 @@ - (void)updateTopViewHorizontalCenterWithRecognizer:(UIPanGestureRecognizer*)rec
     }
 }
 
+- (BOOL)slideToAnchorPosition:(BOOL)isAnchorRight velocity:(CGFloat)velocity translation:(CGFloat)trans {
+    // Invert when we check for the anchor on the left
+    if (!isAnchorRight) {
+        velocity = -velocity;
+        trans = -trans;
```
Comment (kambala-decapitator) [inline XBMC Remote/ECSlidingViewController/ECSlidingViewController.m:291]: please name the variable `xTranslation` or so

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]
you have a typo in the second commit message: tyically

File: XBMC Remote/ECSlidingViewController/ECSlidingViewController.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/ECSlidingViewController/ECSlidingViewController.m]: Done

**Review by kambala-decapitator — APPROVED** [review]

**Comment by wutschel** [comment]
Squashed and force pushed.

**Pull request merged**